schema/annotation-source.yaml

```diff
@@ -147,7 +152,17 @@ $defs:
       An InformationEntity representing an individual piece of data, generated/acquired through methods 
       which reliably produce truthful information about something.
     properties:
-      value:
+      type:
+        type: string
+        const: DataItem
+        default: DataItem
+        description: Must be "DataItem"
+     data_item_type: 
```

Comment:
It may be useful to be able to specify the specific type of data item the value/unit represents.